recipe/meta.yaml

```diff
@@ -2,7 +2,7 @@
 {% set maj_min = ".".join(version.split(".")[:2]) %}
 
 package:
-  name: paraview
+  name: paraview{{ build_variant }}
```

Comment:
Maybe it would make sense to capture this in a metapackage similar to how we handle `blas`, `cuda`, etc.?